theotime2005/bnote-settings
# [MINORFEATURE] Add Bluetooth settings

## Problem

Some Bluetooth settings of the B.note can be adjusted from the interface. In particular, the name, visibility and appearance mode can be adjusted. However, Bluetooth is currently not supported by the interface.

## Solution

- Add a Bluetooth section,

- Define the different parameters,

- Display it on the management interface.

## Note

A new scope has been added, the `text` scope which displays a simple input field.

## Test

- Go to the interface page and display the default file,

- Note the presence of the Bluetooth section,

- Check that it contains the visibility, name and mode of the device,

- Check that the tests pass.
